src/components/messages/InviteMessage/index.tsx

```diff
@@ -56,21 +55,25 @@ const InviteMessage: React.FC<InviteMessageProps> = ({ status }) => {
 
   const StatusEmojis = useMemo(
     () => ({
-      pending: ChatInviteHandEmoji,
-      rejected: HandStopEmoji,
-      approved: ChatInviteHandEmoji
+      pending: '👋',
+      rejected: '✋',
+      approved: '👋'
```

Comment:
Why use these intead of the SVGs isnt there a chance they'll be rendered differently on different machines?

Reply:
Reused the SVG's we had. Could not export these emoji's (👋) in high quality SVG's. Will ask Alex if this is okay.